src/service-worker/core.js

```diff
@@ -71,6 +82,16 @@ function isCdnJsFile(url) {
   );
 }
 
+/**
+ * Determines if a AMP version is blacklisted.
+ * @param {string} version
+ * @return {boolean}
+ */
+function isBlacklisted(version) {
+  // Trim the RTV perfix.
+  version = version.substr(2);
```

Comment:
can we do version.substr(-13) just in case we ever switch the non rtv/ versions to not have the prefix? (i think this was previously the case but not currently)